source/build/ssl_certificates.md

```diff
@@ -24,7 +28,21 @@ We offer a range of certificates from Single Domain, Multiple Domain to Wildcard
 | Extended Validation Multi-Domain SSL |
 +--------------------------------------+
 ```
+- Multi-Domain SSL -  3 domains are included in the standard price, and additional domains can be added at a lower price than a single domain SSL.
+- Unified Communications SSL - Users of Microsoft Exchange 2010/2013 or Office Communications Server 2007 are able to consolidate all certificates into a single, manageable UC Certificate.
+- Extended Validation Multi-Domain SSL -  Like regular EV certificates (see below), EV Multi Domain SSLs also display the green HTTPS website address bar to visitors. 3 domains are included in the standard price and additional domains can be added at a lower price than a single domain SSL.
```

Comment:
Not sure this is true anymore (the green bar) - certainly being moved away from

Reply:
I think I pulled this information from the MyUKFast area when you go to buy an SSL, so it matches what they'll see if they went there.